packages/espressocash_api/lib/src/dto/get_fees.dart

```diff
@@ -10,6 +10,7 @@ class GetFeesResponseDto with _$GetFeesResponseDto {
     required int escrowPayment,
     required int escrowPaymentAtaFee,
     required int splitKeyPayment,
+    required int withdrawalFeePercentage,
```

Comment:
This is a breaking change: until we deploy a new version of the backend, it will fail all the requests. Let's have it as `double withdrawalFeePercentage = 0` for now.